src/transform/src/metadata.ts

```diff
@@ -117,6 +136,10 @@ export class HypermodeMetadata {
     this.functions.forEach((f) => writeItem(f.toString()));
     stream.write("\n");
 
+    writeHeader("Hypermode Embedders:");
```

Comment:
```suggestion
    writeHeader("Embedder Functions:");
```

(just a branding thing)